frontend/src/reducers/card.js

```diff
@@ -10,14 +10,32 @@ const initialState = {
 };
 
 const card = handleActions({
-    [POP_CARD]: (state) => {
+    [POP_CARD]: (state, action) => {
```

Comment:
It looks like we are now are selecting the card by ID, but still removing the card at index 0 no matter what. Fixed in this PR.

https://github.com/RTIInternational/SMART/pull/330

Reply:
Ah I see, you can ignore that PR then.

Reply:
I just pushed a change that only checks if the card at 0 is the right one rather than searching the whole deck